lib/zoom/actions/user.rb

```diff
@@ -102,6 +102,12 @@ def user_token(*args)
         params.require(:id).permit(%i[type ttl])
         Utils.parse_response self.class.get("/users/#{params[:id]}/token", query: params.except(:id), headers: request_headers)
       end
+
+      def user_permissions(*args)
+        params = Zoom::Params.new(Utils.extract_options!(args))
+        params.require(:id).permit(%i[type ttl])
```

Comment:
It just occurred to me, but I am trying to avoid ambiguous `id`s in the codebase. (I'll need to change this for `user_token` above)

```suggestion
        params.require(:user_id).permit(%i[type ttl])
```

Also, where do `type` and `ttl` come from? I don't see this in the docs.

Reply:
ha, those came from copy/paste. I can change for user token in this PR if you like. I also just realized I didnt spec allowing the `type`/`ttl` params in that PR.

Reply:
fwiw, it looks like a lot of the user actions use `id` vs `user_id`

Reply:
I need to change a lot of the actions to use `user_id` haha

Reply:
And yeah, if you wanna change the `user_token` method in this PR that's fine. We can get it shipped out today.

Reply:
Sweet, done

Reply:
hesitant to change existing ones since they'll break the gem's API unless we support both